src/bot_commands.c

```diff
@@ -213,6 +214,29 @@ static void BuildTeamList ()
 	AddTeamToList (&foundTeams, "green", 3, 3);
 }
 
+
+void FrogbotsDumpSkills(void) {
+	for (int skill = 0; skill <= 20; ++skill) {
+		qbool customised_skill = SetAttributesBasedOnSkill (skill);
+		if ( customised_skill ) {
+			G_sprint (self, PRINT_HIGH, "Skipping custom skill %02d\n", skill);
```

Comment:
FWIW if you have custom skill cfgs already in place, running dump won't overwrite them